### ISSUE TITLE
No more stable releases?

### ISSUE THREAD

**Issue by Terminus-IMRC** [crosstool-ng/crosstool-ng#1070]
As mentioned in https://sourceware.org/ml/crossgcc/2018-09/msg00026.html , there are no stable releases since Apr 2017.  We know you are so busy, but do you have a plan to tag a new one?

**Comment by stilor** [comment]
Yes, I plan to tag the next release once I add a few more docker configurations: ubuntu 18.04, ubuntu 16.04, and ubuntu 18.11 (latest non-LTS  release) - and run a build-all in all of them. I also want to run a build-all in cygwin, but that's kind of a "tier-2" target.

I don't have a macos to test, so no "official" support for it in the next release.

**Comment by Terminus-IMRC** [comment]
Glad to hear about that plan.  I'm looking forward to it!

**Comment by bdashore3** [comment]
If you want me to test, I have mac os if needed
